Cargo.toml

```
@@ -4287,3 +4289,15 @@ name = "Widgets"
 description = "Example UI Widgets"
 category = "Helpers"
 wasm = true
+
+[[example]]
+name = "no_std_library"
+path = "examples/no_std/library/src/lib.rs"
+doc-scrape-examples = true
+crate-type = ["lib"]
+
+[package.metadata.example.no_std_library]
+name = "`no_std` Compatible Library"
+description = "Example Library Compatible with `std` and `no_std` Targets"
```

Nitpick: Regular case looks nicer to me and seems more common than title case in other examples.

```suggestion
description = "Example library compatible with `std` and `no_std` targets"
```